project/plugins.sbt

```diff
@@ -12,6 +12,6 @@ addSbtPlugin("uk.gov.hmrc"               % "sbt-distributables" % "2.2.0")
 addSbtPlugin("com.typesafe.play"         % "sbt-plugin"         % "2.8.19")
 addSbtPlugin("org.wartremover"           % "sbt-wartremover"    % "3.0.7")
 addSbtPlugin("org.scalameta"             % "sbt-scalafmt"       % "2.4.0")
-addSbtPlugin("org.scoverage"             % "sbt-scoverage"      % "1.8.0")
+addSbtPlugin("org.scoverage"             %% "sbt-scoverage"     % "1.9.3")
```

Comment:
Thanks, this means that you can remove the scala version override from build jobs